src/templates/app/cashier/index.jsx

```diff
@@ -20,6 +20,12 @@ const DepositWithdraw = ({ id, is_payment_agent, show_upgrade }) => (
     </div>
 );
 
+const CashierNote = ({ text }) => (
+    <div className='invisible client_logged_in cashier-note'>
+        <p>{it.L(text)}</p>
```

Comment:
we can't pass variables to `it.L` in this way. they need to be passed as translated text.